### PR TITLE
Clarifies definitions and adds missing definition

### LINKED ISSUE #34336: curriculum flexbox missing value throughout the challenges for justify-content
the content is missing the space-evenly property for justify content, the curriculum for flexbox may need to be adapted to mention this import value.

### PR DESCRIPTION
**Pull request by Bus42**
Corrects incomplete definitions of space-between space-around values and adds definition of space-evenly value

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #34336

### CODE REVIEW

**Comment by Bus42** [comment]
Hey, so this is my 3rd PR of all time, all 3 of which happened this morning, so I hope I've been helpful!

File: curriculum/challenges/english/01-responsive-web-design/css-flexbox/align-elements-using-the-justify-content-property.english.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Sometimes the flex items within a flex container do not fill all the space in th
...
...ul><li><code>flex-start</code>: aligns items to the start of the flex container. For a row, this pushes the items to the left of the container. For a column, this pushes the items to the top of the container.</li><li><code>flex-end</code>: aligns items to the end of the flex container. For a row, this pushes the items to the right of the container. For a column, this pushes the items to the bottom of the container.</li><li><code>space-between</code>: aligns items to the center of the main axis, with extra space placed between the items. The first and last items are pushed to the very edge of the flex container. For example, in a row the first item is against the left side of the container, the last item is against the right side of the container, then the remaining space is distributed evenly.</li><li><code>space-around</code>: similar to <code>space-between</code> but the first and last items are not locked to the edges of the container, the space is distributed around all the items with a half space on either end of the flex container.</li><li><code>space-evenly</code>: Distributes space evenly between the flex items with a full space at either end of the flex container</li></ul>
```
Comment (ezioda004) [inline curriculum/challenges/english/01-responsive-web-design/css-flexbox/align-elements-using-the-justify-content-property.english.md]: > with a full space at either end of the flex container

The space to the edge is also same as the space b/w the items, you can either omit this part or modify it.

Comment (moT01) [reply curriculum/challenges/english/01-responsive-web-design/css-flexbox/align-elements-using-the-justify-content-property.english.md]: What exactly is problem here, I don't understand?

Comment (RandellDawson) [reply curriculum/challenges/english/01-responsive-web-design/css-flexbox/align-elements-using-the-justify-content-property.english.md]: @ezioda004 What would you like to see fixed on this PR, so we can get it merged?

**Comment by RandellDawson** [comment]
I think we should move forward with merging this PR "as-is".  @ezioda004 If you still find something needs to be changed in the wording, could you create a separate issue, so we can discuss the changes and then create a new PR if the need exists.  Another PR #34419 is waiting for this one which also makes some changes, so after it is merged, this may be a non-issue.

@Manish-Giri If you approve, then I will go ahead and merge it, since we have not heard back from @ezioda004 for over 12 days.

**Review by RandellDawson — APPROVED** [review]
LGTM

**Comment by Manish-Giri** [comment]
@RandellDawson LGTM. Good to merge.

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**